tensorboard/tools/import_google_fonts.py

```diff
@@ -74,7 +74,7 @@
                        'Chrome/62.0.3202.94 '
                        'Safari/537.36',
                        'HTTP User-Agent header to send to Google Fonts')
-flags.DEFINE_string('mirror', 'https://mirror.bazel.build/',
+flags.DEFINE_string('mirror', 'https://mirror.tensorflow.org/',
```

Comment:
Should this url be `http` instead of `https`?

Reply:
Yes! Thank you.